src/components/Shared/RevisionSearch.tsx

```diff
@@ -17,23 +18,23 @@ import SearchResultsList from '../Search/SearchResultsList';
 
 function RevisionSearch(props: RevisionSearchProps) {
   const [focused, setFocused] = useState(false);
-  const { searchResults, view } = props;
+  const { prevRevision, searchResults, setPopoverIsOpen, view } = props;
+
   const dispatch = useAppDispatch();
+  const { replaceSelectedRevision } = useSelectRevision();
 
   const handleFocus = (e: MouseEvent) => {
     if (
       (e.target as HTMLElement).matches(
-        `#search-revision-input, 
-          #search-results-list, 
-          #search-results-list *`,
+        `#revision-search-container, 
+        #revision-search-container *`,
```

Comment:
[question] what's the diff if (we remove the space)
```suggestion
        #revision-search-container*`,
```

Reply:
This is to match any element that is a child of the container element (however far nested). I'm unsure, but I think without the space it either would not work or it would try to match any element with an id that begins with `revision-search-container`